app/controllers/places_controller.rb

```diff
@@ -0,0 +1,21 @@
+class PlacesController < ApplicationController 
+
+  def show
+    byebug
+    place_id = find_id(params[:go_to])
+
+    @client = GooglePlaces::Client.new(ENV['GOOGLE_KEY'])
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.